test-cypress/integration/import/import.server.files.spec.js

```diff
@@ -1,137 +1,72 @@
-        // Should return to import page
-        cy.navigateToPage('Import', 'RDF');
-
-        cy.chooseServerFilesTab();
-        cy.selectDistinctFileToImport(blankNodeFile);
-        cy.importChangingSettings().wait(2000);
-        cy.verifyMessageOccurs(messageSuccess);
-        cy.confirmTextPresentsInDistinctFileInfo('"preserveBNodeIds": false,', blankNodeFile);
-
-        // First navigate to create repository
-        cy.navigateToPage('Setup', 'Repositories');
-        cy.deleteRepo(repoId).wait(1000);
-        cy.selectRepo(existingRepo).wait(1000);
-        // Should return to import page
-        cy.navigateToPage('Import', 'RDF');
-        cy.chooseServerFilesTab();
+    it('Test import all Server files successfully without changing settings', () => {
+        ImportSteps
+            .selectAllServerFiles()
+            .importServerFiles()
+            .verifyImportStatus(fileToImport, messageSuccess)
+            .verifyImportStatus(blankNodeFile, messageSuccess);
     });
 
-
-    it('Test import Server file successfully with preserve BNodes IDs = true', function () {
-        let repoId = 'repo3' + Date.now();
-        // First navigate to create repository
-        cy.navigateToPage('Setup', 'Repositories');
-        // Create new one
-        cy.createNewRepo(repoId).wait(2000);
-        cy.selectRepo(repoId).wait(1000);
-        // Should return to import page
-        cy.navigateToPage('Import', 'RDF');
-
-        cy.chooseServerFilesTab();
-
-        cy.selectDistinctFileToImport(blankNodeFile);
-        cy.importChangingSettings(true);
-        cy.expandAdvancedSettings();
-        cy.get('[name="preserveBNodeIDs"]').check();
-        cy.clickImportBtnOnPopUpMenu().wait(1500);
-        cy.verifyMessageOccurs(messageSuccess);
-        cy.confirmTextPresentsInDistinctFileInfo('"preserveBNodeIds": true,', blankNodeFile);
-
-        // First navigate to create repository
-        cy.navigateToPage('Setup', 'Repositories');
-        cy.deleteRepo(repoId).wait(1000);
-        cy.selectRepo(existingRepo).wait(1000);
-        // Should return to import page
-        cy.navigateToPage('Import', 'RDF');
-        cy.chooseServerFilesTab();
+    it('Test import with resetting status of imported file', () => {
+        ImportSteps
+            .selectServerFile(fileToImport)
+            .importServerFiles()
+            .verifyImportStatus(fileToImport, messageSuccess)
+            .resetStatusOfUploadedFile(fileToImport)
+            .verifyNoImportStatus(fileToImport);
     });
 
-    it('Test import all Server files successfully without changing settings', function () {
-        cy.chooseServerFilesTab();
-        cy.checkAllServerFiles(true);
-        cy.importChangingSettings().wait(2000);
-        cy.verifyMessageOccurs(messageSuccess);
+    it('Test import with resetting status of all imported files', () => {
+        ImportSteps
+            .selectAllServerFiles()
+            .importServerFiles()
```

Comment:
There is a lot of room for improvement. For example these two can be combined under single function +  `verifyNoImportStatus` can receive array of file names.